orderable/tests/test_manager.py

```diff
@@ -0,0 +1,66 @@
+from django.test import TestCase
+
+from .models import SubTask, Task
+
+
+class TestOrderableManager(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.tasks = []
+        for i in range(3):
+            task = Task(sort_order=i)
+            task.save()
+            cls.tasks.append(task)
+
+    def test_gets_next(self):
+        first_task = self.tasks[0]
+        next_task = Task.objects.after(first_task)
+        self.assertEqual(next_task, self.tasks[1])
```

Comment:
Or, if you add a factory:

```
self.first_task, self.middle_task, self.last_task = TaskFactory.create_batch(3)
```

(although I don't know if that'll fit on one line :P)